cudaaligner/benchmarks/main.cpp

```diff
@@ -80,6 +80,8 @@ typename std::enable_if<!std::is_same<AlignerT, AlignerGlobalMyersBanded>::value
 template <typename AlignerT>
 typename std::enable_if<std::is_same<AlignerT, AlignerGlobalMyersBanded>::value, std::unique_ptr<Aligner>>::type create_aligner_tmp_dispatch(int32_t genome_size, int32_t alignments_per_batch, DefaultDeviceAllocator allocator, cudaStream_t stream, int32_t device_id)
 {
+    (void)genome_size;
```

Comment:
It does not make much difference, but you can use C++ cast: `static_cast<void>(genome_size);`

Reply:
addressed